.github/workflows/buf.yaml

```diff
@@ -0,0 +1,16 @@
+on:
+  push:
+    branches:
+      - master
```

Comment:
I think we want to run the lint and breaking change detector on all PRs, and just limit `push` to the `master` branch.